zerver/lib/actions.py

```diff
@@ -852,6 +852,9 @@ def do_change_realm_subdomain(realm: Realm, new_subdomain: str) -> None:
     realm.save(update_fields=["string_id"])
 
 def do_scrub_realm(realm: Realm, acting_user: Optional[UserProfile]=None) -> None:
+    if settings.BILLING_ENABLED:
+        downgrade_now_without_creating_additional_invoices(realm)
```

Comment:
Hmm makes me think, should we run some db queries on prod to scan for realms that might have gotten scrubbed but still have an active plan? (as unlikely as that seems)

Reply:
Yeah. It's worth checking that out.

```python
from zerver.models import RealmAuditLog
from corporate.models import get_current_plan_by_realm

def check_scrubbed_realms_with_active_plan_exist():
    audit_logs = RealmAuditLog.objects.filter(event_type=RealmAuditLog.REALM_SCRUBBED)
    for log in audit_logs:
        if get_current_plan_by_realm(log.realm):
            print(log.realm.string_id)
````

@alexmv Can you try running this script and see if any realms present that are scrubbed but still have an active plan?

Reply:
Sorry for dropping the ball on this!

There's no output in production currently.